packages/client/components/DashTopBar.tsx

```diff
@@ -48,11 +54,21 @@ const LeftNavHeader = styled('div')({
   width: NavSidebar.WIDTH
 })
 
-const Img = styled('img')({
+const LogoWrapper = styled('button')({
+  background: 'transparent',
+  border: 'none',
+  borderRadius: 4,
   cursor: 'pointer',
-  margin: '8px 16px'
+  margin: '8px 0',
+  padding: '8px 8px 4px 8px',
+  ':focus': {
+    boxShadow: '0 0 0 2px #61B1EB',
+    outline: 'none'
+  }
 })
 
+const Img = styled('img')({})
```

Comment:
You can remove this and use a simple `<img` below instead of a styled component `<Img